backend/substrapp/tests/views/tests_views_algo.py

```diff
@@ -204,14 +204,19 @@ def test_algo_create(self):
                 'objective_key': get_hash(os.path.join(
                     dir_path, '../../../../fixtures/chunantes/objectives/objective0/description.md')),
                 'permissions_public': True,
-                'permissions_authorized_ids': []}
+                'permissions_authorized_ids': [],
+                'metadata': {
+                    "foo": "bar",
+                    "bar": "foo"
+                }}
 
         with mock.patch.object(LedgerAlgoSerializer, 'create') as mcreate:
 
             mcreate.return_value = {}
 
             response = self.client.post(url, data=data, format='multipart', **self.extra)
 
+        assert response.data['metadata'] is not None
```

Comment:
Since you know what the metadata must be, you could assert that:

```python
assert response.data['metadata'] ==  {
                    "foo": "bar",
                    "bar": "foo"
                }
```